### DISCUSSION TITLE
Composables not available

### DISCUSSION THREAD

**Discussion by Gyurmatag** [nuxt/nuxt#17777] [Questions]
In the latest Nuxt Version: 3.0.0-27480376.fdd38f9 I am not able to reference my const from my composables without importing it.
Here is composables/apiUris.ts:
```
export const AUTH = "auth";
export const IS_EMAIL_ALREADY_REGISTERED = "is-email-already-registered";
export const SIGNIN = "signin";
export const SIGNUP = "signup";
export const RESERVATION = "reservation";
export const SAVE = "savąe";
export const USER_RESERVATIONS = "user-reservations";
```

In one of my component I try to access the constants like this:
```
const { data: userData, error } = await useCustomFetch(
        `${AUTH}/${SIGNIN}`,
        {
          method: "POST",
          body: {
            email: formData.value.email,
            password: formData.value.password,
          },
        }
      );
```

I've got this error in my browser console:
`Uncaught (in promise) ReferenceError: AUTH is not defined
`
If I import from composables it's working, but composables should be auto-imported.
What is the problem here?

Thanks very much!

**Comment by mihail727** [comment]
Interesting problem - maybe you need to open an issue...
To solve this problem, you can try to wrap all your constants in an object
```ts
export const useConst = {
	AUTH: 'auth',
	IS_EMAIL_ALREADY_REGISTERED: 'is-email-already-registered',
	SIGNIN: 'signin',
	SIGNUP: 'signup',
	RESERVATION: 'reservation',
	SAVE: 'savąe',
	USER_RESERVATIONS: 'user-reservations',
};
```
